search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -171,4 +162,44 @@ class TemporalEntityHandler(
         return buildGetSuccessResponse(mediaType, contextLink)
             .body(serializeObject(addContextsToEntity(temporalEntity, listOf(contextLink), mediaType)))
     }
+
+    /**
+     * Implements 5.6.15 - Delete Attribute instance from Temporal Representation of an Entity
+     */
+    @DeleteMapping("/{entityId}/attrs/{attrId}/{instanceId}")
+    suspend fun deleteAttributeInstanceTemporal(
+        @RequestHeader httpHeaders: HttpHeaders,
+        @PathVariable entityId: String,
+        @PathVariable attrId: String,
+        @PathVariable instanceId: String
+    ): ResponseEntity<*> {
+        return either<APIException, ResponseEntity<*>> {
+            val sub = getSubFromSecurityContext()
+            val entityUri = entityId.toUri()
+            val instanceUri = instanceId.toUri()
+            val contexts = listOf(getContextFromLinkHeaderOrDefault(httpHeaders))
+            val expandedAttrId = JsonLdUtils.expandJsonLdTerm(attrId, contexts)!!
+
+            temporalEntityAttributeService.checkEntityExistence(entityUri).bind()
+
+            val temporalEntityAttributeUUID =
+                temporalEntityAttributeService.checkEntityAttributeExistence(entityUri, expandedAttrId).bind()
```

Comment:
this could be done in one single request

Reply:
maybe you could also use the `getForEntityAndAttribute` function which is almost the same as the function(s) you've added

Reply:
ok but i have no way to know which between entityid and attributename does not exist ?

Reply:
deal with it in the error message :)

Reply:
Ok I look this